test/run

```diff
@@ -9,7 +9,7 @@ usage() {
     echo "Unrecognized options are passed through to dejagnu by default."
     echo
     echo "Interesting options:"
-    echo "  --tool_exec=    Test against a different bash executable."
+    echo "  --tool=    Test against a different bash executable."
```

Comment:
Please add some space to align the description with the rest (can't see it, but I suppose we're not using tabs there).